etc/scripts/d2d/README.rst

```diff
@@ -0,0 +1,92 @@
+Run ScanCode.io Mapping Script
+================================
+
+This script executes the ``map_deploy_to_develop`` mapping workflow from
+ScanCode.io inside a Docker container. It optionally spins up a temporary
+PostgreSQL instance when needed. The script copies the specified input files to
+a working directory, runs the mapping, writes the output to a file, and cleans
+up afterward.
+
+Usage
+-----
+
+.. code-block:: bash
+
+   ./map-deploy-to-develop.sh <from-path> <to-path> <output-file> [options] <spin-db> [db-port]
+
+Arguments
+---------
+
++-----------------+-------------------------------------------------------------+
+| Argument        | Description                                                 |
++=================+=============================================================+
+| ``from-path``   | Path to the base deployment/scan file                       |
++-----------------+-------------------------------------------------------------+
+| ``to-path``     | Path to the target deployment/scan file                     |
++-----------------+-------------------------------------------------------------+
+| ``options``     | D2D pipeline parameters (can be empty ``""``)               |
```

Comment:
`options` should also be a bit more descriptive, to communicate that these are ecosystem specific optional steps, and instead of `can be empty` we can probably mention in someway which parameters are optional and which are required.

We should probably also have a reference page on all the supported ecosystems in d2d and the capabilities supported there, and link to this page. I will open a seperate issue for this.